content/docs/integrations/nginx_ingress_controller.md

```diff
@@ -0,0 +1,121 @@
+---
```

Comment:
This workflow is documented as a part of the sample demo here: https://docs.openservicemesh.io/docs/tasks_usage/traffic_management/ingress/#sample-demo